packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -158,7 +159,7 @@ const AppNotificationMessages = ({
     const getNotificationSublist = () => {
         if (window.location.pathname === routes.cashier_deposit) {
             return filtered_excluded_notifications.filter(message =>
-                message.key.includes('switched_to_real') ? message : null
+                message.key.includes('switched_to_real', 'failed_wallets_upgrade') ? message : null
```

Comment:
@aizad-deriv I believe the `Array.includes` don't accept multiple arguments, you can try the following,
let me know if it works 👍 

```suggestion
                message.key.some(msg => msg === 'switched_to_real' || msg === 'failed_wallets_upgrade') ? message : null
```

Reply:
Hi @sandeep-deriv I ended up removing the key, `failed_wallets_upgrade` it seems that I do not need it because it is supposed to show on the deposit page. But after applying the changes you suggested it seems that it is giving an error, because the `switched_to_real` is not used anymore. That being said, I genuinely think this should be handled outside this PR because this is a `feature branch`. 
Will revert this back to where it was previously.